src/content/test/errors/status-messages.tsx

```diff
@@ -0,0 +1,162 @@
+            `}
+        />
+
+        <h2>More examples</h2>
+
+        <h3>WCAG success criteria</h3>
+        <Markup.Links>
+            <Markup.HyperLink href="https://www.w3.org/WAI/WCAG21/Understanding/status-messages.html">
+                Understanding Success Criterion 4.1.3: Status Messages
+            </Markup.HyperLink>
+        </Markup.Links>
+
+        <h3>Sufficient techniques</h3>
+        <h4>If a status message conveys a suggestion, or a warning on the existence of an error:</h4>
+        <Markup.Links>
+            <Markup.HyperLink href="https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA19">
+                Using ARIA role=alert or Live Regions to Identify Errors
+            </Markup.HyperLink>
+        </Markup.Links>
+        <p>in combination with any of the following:</p>
+        <Markup.Links>
+            <Markup.HyperLink href="https://www.w3.org/WAI/WCAG21/Techniques/general/G83">
+                Providing text descriptions to identify required fields that were not completed
+            </Markup.HyperLink>
+            <Markup.HyperLink href="https://www.w3.org/WAI/WCAG21/Techniques/general/G84">
+                Providing a text description when the user provides information that is not in the list of allowed values
+            </Markup.HyperLink>
+            <Markup.HyperLink href="https://www.w3.org/WAI/WCAG21/Techniques/general/G85">
+                Providing a text description when user input falls outside the required format or values
+            </Markup.HyperLink>
+            <Markup.HyperLink href="https://www.w3.org/WAI/WCAG21/Techniques/general/G177">
+                Providing suggested correction text
+            </Markup.HyperLink>
+            <Markup.HyperLink href="https://www.w3.org/WAI/WCAG21/Techniques/general/G194">
+                Providing spell checking and suggestions for text input
+            </Markup.HyperLink>
+        </Markup.Links>
+
+        <p>If a status message advises on the success or results of an action, or the state of an application:</p>
+        <Markup.Links>
+            <Markup.HyperLink href="https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA22">
+                Using role=status to present status messages
+            </Markup.HyperLink>
+        </Markup.Links>
+        <p>in combination with:</p>
+        <Markup.Links>
+            <Markup.HyperLink href="https://www.w3.org/WAI/WCAG21/Techniques/general/G199">
+                Providing success feedback when data is submitted successfully
+            </Markup.HyperLink>
+        </Markup.Links>
+
+        <p>If a status message conveys information on the progress of a process</p>
+        <Markup.Links>
+            <Markup.HyperLink href="https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA23">
+                Using role=log to identify sequential information updates
+            </Markup.HyperLink>
+        </Markup.Links>
+        <p>Using role="progressbar" (future link)</p>
```

Comment:
Is the `(future link)` text here (and in a few other places, eg L155/L159) intended to have been replaced with an actual link at this point?

Reply:
I think you're right; good question for @ferBonnin.

Reply:
It is a placeholder that the W3C content uses too. for this line, we can remove it because it now works. For L155/159, there are no links yet. we could either remove that whole section or leave it whitout links

Reply:
Edit: there are no links for any of this future links...

Reply:
The W3C has them as is, but I am unsure how helpful it is to list this without those links. Thoughts? @dbjorge @waabid

Reply:
I think there's still some value in listing them as descriptive examples even without more detailed links. I'd lean towards removing the ` (future link)` text but otherwise leaving it as is (for all 3 cases)

Reply:
I will remove them.